kairon/shared/account/processor.py

```
@@ -959,6 +959,7 @@ def load_system_properties():
                 ).read(),
                 button_template=open("template/emails/button.html", "r").read(),
                 leave_bot_owner_notification=open("template/emails/leaveBotOwnerNotification.html", "r").read(),
+                catalog_sync_status=open("template/emails/catalog_sync_status.html", "r").read(),
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)